boa/schemas/model.py

```diff
@@ -0,0 +1,97 @@
+# generated by datamodel-codegen:
+#   filename:  recipe.v1.json
+#   timestamp: 2021-11-22T15:42:45+00:00
+
+from __future__ import annotations
+
+from typing import Any, Dict, List, Optional
+
+from pydantic import BaseModel, Extra, Field, conint
+
+
+class Package(BaseModel):
+    class Config:
+        extra = Extra.forbid
+
+    name: Optional[str] = Field(None, description="The package name")
+    version: Optional[str] = Field(None, description="The package version")
```

Comment:
as well as version should also be mandatory

Reply:
done